### ISSUE TITLE
Configurable tailwind.config.js and node_modules paths

### ISSUE THREAD

**Issue by mlvzk** [tailwindlabs/tailwindcss-intellisense#249]
I have a mono-repo where I have a frontend and backend directory. When I open vscode I open it in the mono-repo directory, not frontend. To make this extension work I had to make a symbolic link from ./frontend/node_modules and ./frontend/tailwind.config.js to ./node_modules and ./tailwind.config.js. This wouldn't work on Windows, and although I won't be using Windows to develop this project, I wish I didn't have to use this hack.

Both a high-level configuration key like "tailwindCSS.workspaceDir" and more low level "tailwindCSS.nodeModulesDir", "tailwindCSS.configPath" keys would be fine with me.

Would you consider adding this feature?

**Comment by bradlc** [comment]
Hey @mlvzk. To clarify your project looks something like this:

```
my-project/
├── backend/
└── frontend/
    ├── node_modules/
    └── tailwind.config.js
```

And you are opening the `my-project` folder in VS Code? In that case the extension should work fine. The config file and `node_modules` folder do not need to be at the project root. Please can you open the VS Code `Output` panel and look for any logs under `tailwindcss-intellisense` (please remove your symlinks and reload VS Code first)

![image](https://user-images.githubusercontent.com/2615508/105482647-93da1d80-5ca0-11eb-94a1-56e66a935948.png)

**Comment by mlvzk** [comment]
Hello @bradlc
Yes, that's my project structure. I should also add that I have node_modules in my backend/ directory as well.

I tried again after removing the symbolic links and it seems to be working, which is very strange. I thought maybe it was picking up node_modules from my backend/ directory but after closing and opening vscode multiple times that doesn't seem to be the case.

Closing the issue for now, will re-open when it happens again. Thanks for your response.
